vidarr-cli/src/main/java/ca/on/oicr/gsi/vidarr/cli/TestValidatorScript.java

```diff
@@ -19,15 +19,22 @@ public final class TestValidatorScript extends TestValidator {
   private String outputMetrics;
 
   @Override
-  public Validator createValidator() {
+  public Validator createValidator(String outputDirectory, String id) {
     try {
-      final var tempDir = Files.createTempDirectory("vidarr-test-script");
-      final var calculateScript = tempDir.resolve("calculate");
-      final var calculateDir = tempDir.resolve("output");
-      Files.createDirectories(calculateDir);
-      Files.copy(Path.of(metricsCalculate), calculateScript);
+      // If output directory provided, use that else we create temporary directory /tmp
+      final var finalDir = (outputDirectory != null) ? Path.of(outputDirectory)
+          : Files.createTempDirectory("vidarr-test-script");
+      final var finalCalculateScript =
+          (outputDirectory != null) ? Path.of(outputDirectory, id) : finalDir.resolve("calculate");
```

Comment:
This is the path to the calculate script that vidarr-cli will call out to (line 86). Users should name this file after the configuration id instead, if they want to specify an output path?

Reply:
This issue is still outstanding, and to clarify my comment, I am not asking, this behaviour should not happen.